src/ui/components/Library/Navigation/TeamButton.tsx

```diff
@@ -43,25 +46,62 @@ export function TeamButton({
   };
 
   return (
-    <Link
-      href={url}
+    <div
       className={classNames(
-        `${styles.teamRow} group flex flex-row justify-between space-x-2 px-4 py-2 text-left transition duration-200 hover:text-white focus:outline-none`,
-        isSelected ? `${styles.teamRowActive} cursor-auto` : "cursor-pointer"
+        `flex flex-col ${styles.teamRow} `,
+        isSelected ? `${styles.teamRowActive}` : ""
       )}
-      onClick={onClick}
     >
-      <span className="overflow-hidden overflow-ellipsis whitespace-pre">
-        <div className="flex">
-          <LibraryIcon teamType={isTest ? "tests" : "team"} />
-          {label}
+      <Link
+        href={url}
+        className={classNames(
+          `group flex flex-row justify-between space-x-2 px-4 py-2 text-left transition duration-200 hover:text-white focus:outline-none`,
+          isSelected ? `cursor-auto` : "cursor-pointer"
+        )}
+        onClick={onClick}
+      >
+        <span className="overflow-hidden overflow-ellipsis whitespace-pre">
+          <div className="flex">
+            <LibraryIcon teamType={isTest ? "tests" : "team"} />
+            {label}
+          </div>
+        </span>
+        {isNew ? (
+          <div className={"rounded-md bg-primaryAccent px-3 py-0.5 text-xs text-white"}>New</div>
```

Comment:
This is a pretty wide button. Testing it I realized that we don't really handle text overflow well in the side nav:
<img width="264" alt="Screen Shot 2023-11-16 at 8 10 34 AM" src="https://github.com/replayio/devtools/assets/29597/83463949-cc5f-474b-9677-124f4fb14373">

I think the above style was meant to address that, but doesn't:
```html
<span className="overflow-hidden overflow-ellipsis whitespace-pre">
```

I think to fix it we'd want to move that wrapper around the `{label}`:
```jsx
<div className="flex overflow-hidden">
  <LibraryIcon teamType={isTest ? "tests" : "team"} />
  <span className="overflow-hidden overflow-ellipsis whitespace-pre">{label}</span>
</div>
```